src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -181,16 +185,21 @@ private static String getReasonForBeingLate(SyaCaseWrapper syaCaseWrapper) {
     }
 
     private static String getDwpIssuingOffice(SyaCaseWrapper syaCaseWrapper) {
+        DwpAddressLookupService dwpLookup = new DwpAddressLookupService();
         String benefitType = syaCaseWrapper.getBenefitType().getCode();
+
         if (benefitType.equalsIgnoreCase("uc")
```

Comment:
It would be nice to change this condition also to UC == Benefit.getBenefitByCode(benefitType)